daemon/rename.go

```diff
@@ -121,15 +122,20 @@ func (daemon *Daemon) ContainerRename(oldName, newName string) (retErr error) {
 			}
 		}()
 
+		sbEps := make(map[string]*libnetwork.Endpoint)
+		for _, ep := range sb.Endpoints() {
+			sbEps[ep.Network()] = ep
+		}
+
 		for nwName, epConfig := range container.NetworkSettings.Networks {
 			nw, err := daemon.FindNetwork(nwName)
 			if err != nil {
 				return err
 			}
 
-			ep, err := nw.EndpointByID(epConfig.EndpointID)
```

Comment:
`Sandbox` has an unexported method `getEndpoint(id string)`; why not export and use that?